Gemfile

```diff
@@ -28,6 +28,10 @@ gem 'sdoc', '~> 0.4.0', group: :doc
 # Discussion: https://github.com/18F/college-choice/issues/597#issuecomment-139034834
 gem "puma", "~> 2.16.0"
 
+#redis and redis-namespace for session and mvi persistence
+gem 'redis'
```

Comment:
This is just the redis gem though, not Redis. According to docs, the latest version supports Redis 2.0+